docs/website/contents/for-developers/HandlingBlocksFromTheFuture.md

```diff
@@ -0,0 +1,44 @@
+#  Blocks from the future
+
+A node can receive a block whose slot is ahead of the current slot. We call such **blocks from the future**.
+
+The Praos protocol ignores chains with blocks from the future during chain selection.
+It assumes nodes have perfectly synchronized clocks, which is not realistic due to imperfections in protocols like NTP and leap seconds.
+In practice, a clock skew of up to 5 seconds is considered acceptable.
```

Comment:
This has now been reduced to 2 seconds, see https://github.com/IntersectMBO/ouroboros-consensus/pull/1269#discussion_r1792230276 and #1279.